### PR TITLE
Warn users about application type mismatch

### PR DESCRIPTION
**Pull request by oleksandrpravosudko-okta**
## Description:
- **What's changed?** 
- Note block quote is added to guides mentioning Okta app creation
- **Is this PR related to a Monolith release?** 
No

### Resolves:

* [OKTA-342950](https://oktainc.atlassian.net/browse/OKTA-342950)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/implement-auth-code-pkce/setup-app/index.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ You set up your OpenID Connect application inside the Okta Developer Console:
 
 1. From the Applications page, choose **Add Application**.
 2. On the Create New Application page, select **Native** or **SPA**, depending on the type of application you are working on.
+  > **Note:** It is important to choose appropirate application type for apps which are public clients. Failing to do so may result in Okta API endpoints attempting to verify app's client secret, which public clients are not designed to have, hence breaking sign in or sign out flow.
```
Comment (jakubvul-okta) [inline packages/@okta/vuepress-site/docs/guides/implement-auth-code-pkce/setup-app/index.md:9]:
```suggestion
  > **Note:** It is important to choose the appropriate application type for apps which are public clients. Failing to do so may result in Okta API endpoints attempting to verify an app's client secret, which public clients are not designed to have, hence breaking the sign-in or sign-out flow.
```

File: packages/@okta/vuepress-site/docs/guides/implement-implicit/setup-app/index.md [hunk 2]
```diff
@@ -6,6 +6,7 @@ You set up your OpenID Connect application inside the Okta Developer Console:
 
 1. From the Applications page, choose **Add Application**.
 2. On the Create New Application page, select **SPA**.
+  > **Note:** It is important to choose appropirate application type for apps which are public clients. Failing to do so may result in Okta API endpoints attempting to verify app's client secret, which public clients are not designed to have, hence breaking sign in or sign out flow.
```
Comment (jakubvul-okta) [inline packages/@okta/vuepress-site/docs/guides/implement-implicit/setup-app/index.md:9]:
```suggestion
  > **Note:** It is important to choose the appropriate application type for apps which are public clients. Failing to do so may result in Okta API endpoints attempting to verify an app's client secret, which public clients are not designed to have, hence breaking the sign-in or sign-out flow.
```

File: packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta/create-oauth-app/index.md [hunk 3]
```diff
@@ -10,6 +10,7 @@ Create the client application that you want to use with the Okta APIs.
 2. Click **Applications** from the menu, and then click **Add Application**.
 
 3. On the **Create New Application** page, select any app type and then click **Next**. We suggest creating a web, single-page, or native app for an easy way to test scope-based access to Okta's APIs using an OAuth 2.0 bearer token.
+  > **Note:** It is important to choose appropirate application type for apps which are public clients. Failing to do so may result in Okta API endpoints attempting to verify app's client secret, which public client are not designed to have, hence breaking sign in or sign out flow.
```
Comment (jakubvul-okta) [inline packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta/create-oauth-app/index.md:13]:
```suggestion
  > **Note:** It is important to choose the appropriate application type for apps which are public clients. Failing to do so may result in Okta API endpoints attempting to verify an app's client secret, which public clients are not designed to have, hence breaking the sign-in or sign-out flow.
```

File: packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app/create-okta-application/index.md [hunk 4]
```diff
@@ -8,6 +8,7 @@ First, sign in to the Okta Developer Console:
 <a href="https://login.okta.com/" target="_blank" class="Button--blue">Go to Console</a>
 
 1. Select **Applications**, then **Add Application**. Pick **Native** as the platform. Enter a name for your application (or leave the default value).
+  > **Note:** It is important to choose appropirate application type for apps which are public clients. Failing to do so may result in Okta API endpoints attempting to verify app's client secret, which public client are not designed to have, hence breaking sign in or sign out flow.
```
Comment (jakubvul-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app/create-okta-application/index.md:11]:
```suggestion
  > **Note:** It is important to choose the appropriate application type for apps which are public clients. Failing to do so may result in Okta API endpoints attempting to verify an app's client secret, which public clients are not designed to have, hence breaking the sign-in or sign-out flow.
```

File: packages/@okta/vuepress-site/docs/guides/sign-into-spa/create-okta-application/index.md [hunk 5]
```diff
@@ -8,6 +8,7 @@ Start by signing in to the Okta Developer Console:
 <a href="https://login.okta.com/" target="_blank" class="Button--blue">Go to Console</a>
 
 1. Select **Applications**, then **Add Application**. Pick **Single-Page App (SPA)** as the platform. Enter a name for your application (or leave the default value).
+> **Note:** It is important to choose appropirate application type for apps which are public clients. Failing to do so may result in Okta API endpoints attempting to verify app's client secret, which public client are not designed to have, hence breaking sign in or sign out flow.
```
Comment (jakubvul-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-into-spa/create-okta-application/index.md:11]:
```suggestion
  > **Note:** It is important to choose the appropriate application type for apps which are public clients. Failing to do so may result in Okta API endpoints attempting to verify an app's client secret, which public clients are not designed to have, hence breaking the sign-in or sign-out flow.
```

**Review by jakubvul-okta — CHANGES_REQUESTED** [review]

**Comment by aarongranick-okta** [comment]
@oleksandrpravosudko-okta please fix spelling of "appropirate" (yeargh!)

**Comment by oleksandrpravosudko-okta** [comment]
@aarongranick-okta  there seem to be no more ~seadogs~ "appropirate" occurrances left unfixed

**Review by jakubvul-okta — APPROVED** [review]

**Pull request merged**